docs/rules/require-default-props.md

```diff
@@ -165,7 +165,7 @@ MyStatelessComponent.propTypes = {
 };
 
 MyStatelessComponent.defaultProps = {
-    bar: 'some default'
```

Comment:
let's also fix the indentation in the previous example

Reply:
Where is typo in the previouse example? I don't find that. It's means another document?

Reply:
the previous example seems to have one-space indentation

Reply:
Sorry, I don't understand. Are you talking about revert?

Reply:
I'm talking about line 165 - it looks like the `};` is an extra space indented. is that just github's UI?

Reply:
There is no extra space indented. Just github's UI :)